datadotworld/client/_swagger/models/file_create_or_update_request.py

```diff
@@ -21,32 +21,52 @@ class FileCreateOrUpdateRequest(object):
     NOTE: This class is auto generated by the swagger code generator program.
     Do not edit the class manually.
     """
-    def __init__(self, name=None, source=None):
+
+
+    """
+    Attributes:
+      swagger_types (dict): The key is attribute name
+                            and the value is attribute type.
+      attribute_map (dict): The key is attribute name
+                            and the value is json key in definition.
+    """
+    swagger_types = {
+        'name': 'str',
+        'source': 'FileSourceCreateOrUpdateRequest',
+        'description': 'str',
+        'labels': 'list[str]'
+    }
+
+    attribute_map = {
+        'name': 'name',
+        'source': 'source',
+        'description': 'description',
```

Comment:
`description` and `labels` need to be exposed in the respective `api.py` wrapper.